### ISSUE TITLE
HA 0.91.3 cron settings icon is no longer display for name editing after unique_id is added for each component

### ISSUE THREAD

**Issue by jerrychong25** [home-assistant/home-assistant-polymer#3091]
**Home Assistant release with the issue:**
HA 0.91.3

**Last working Home Assistant release (if known):**
0.90.2

**UI (States or Lovelace UI?):**
Lovelace UI

**Browser and Operating System:**
Chrome Version 73.0.3683.103 (Official Build) (64-bit)
macOS Majove Version 10.14.4

**Description of problem:**
In 0.90.2, after adding **unique_id** for light component, it will have cron settings icon displayed in frontend for user to edit the display name, as per screenshot below:
![63f5ca668f52fcba63a014856883b5c9bb9bab60](https://user-images.githubusercontent.com/15714095/56107569-9959ee00-5f79-11e9-8bb0-2e5ee697139b.jpeg)

In 0.91.3, it does not displayed this cron settings icon anymore after I upgraded HA OS without changing any settings, as per screenshot below:
![d9f3fc1a883fca6bce2fcdb160ce56b454b29fd6](https://user-images.githubusercontent.com/15714095/56107589-bb537080-5f79-11e9-805c-002be7332d34.jpeg)

**Javascript errors shown in the web inspector (if applicable):**
N/A

**Additional information:**
Could advise how to display the cron settings icon on HA 0.91.3?

As changing the display name in backend is time consuming, which need to restart HA every time after each edit is done.

Thanks.

**Source code:**
```
# Configuration.yaml
light:
- platform: mqtt
  name: 'light_1'
  unique_id: 'light_1'
```

**Comment by jerrychong25** [comment]
Any updates on this issue?

Thanks.

**Comment by jerrychong25** [comment]
Source: https://www.home-assistant.io/docs/configuration/customizing-devices/

<img width="724" alt="Screenshot 2019-04-27 at 2 35 20 PM" src="https://user-images.githubusercontent.com/15714095/56845824-bedfe380-68f9-11e9-879e-611be38b6869.png">

```
Changing the entity_id
You can use the UI to change the entity_id and friendly name of supported entities. To do this:

Select the entity, either from or from the frontend, or by clicking  next to the entity in the dev-states menu
Click on the cog in the right corner of the entity’s dialog
Enter the new name or the new entity ID (remember not to change the domain of the entity - the part before the .)
Select Save
```

**Comment by jerrychong25** [comment]
This issue is fixed after upgraded to latest HA version 0.92.0.
